recipes/ignition-cmake/all/conanfile.py

```diff
@@ -39,6 +39,9 @@ def source(self):
         os.rename(glob.glob("ign-cmake*")[0], self._source_subfolder)
 
     def build(self):
+        version_major = tools.Version(self.version).major
+        ignition_cmake_dir = os.path.join(self.package_folder, "lib", "cmake", f"ignition-cmake{version_major}", "cmake2")
+        os. environ['IGNITION_CMAKE_DIR'] = f"{ignition_cmake_dir}"
```

Comment:
Is it environment variable needed during cmake execution? os.environ is not a good option, prefer https://docs.conan.io/en/latest/reference/build_helpers/run_environment.html#runenvironment

Reply:
Was not sure about this. Thanks for feedback. I removed it